draft-ietf-iotops-7228bis.md

```diff
@@ -163,26 +155,30 @@ and capabilities are becoming connected.  Mobile personal gadgets,
 building-automation devices, cellular phones, machine-to-machine (M2M)
 devices, and other devices benefit from interacting with other "things" nearby
 or somewhere in the Internet.  With this, the Internet of Things (IoT)
-becomes a reality, built up out of uniquely identifiable and
-addressable objects (things).  Over the next decade, this could
-grow to large numbers of Internet-connected constrained
-devices ({{IoT-2025}} predicts that by, 2025, more than
-2500 devices will be connected to the Internet per second), greatly
-increasing the Internet's size and scope.
+became a reality, built up out of uniquely identifiable and
+addressable objects (things).
 
 The present document provides a number of basic terms that have
-been useful in the standardization work for constrained
+been useful in research and standardization work for constrained
 environments.  The intention is not to exhaustively cover the field
 but to make sure a few core terms are used consistently between
 different groups cooperating in this space.
 
 The present document is a revision of {{RFC7228}}.
 
+## Conventions Used in this Document
+
 In this document, the term "byte" is used in its now customary sense
 as a synonym for "octet".  Where sizes of semiconductor memory are
 given, the prefix "kibi" (1024) is combined with "byte" to "kibibyte",
 abbreviated "KiB", for 1024 bytes {{ISQ-13}}.
-Powers of 10 are given as 10<sup>100</sup> where 100 is the exponent.
+
+Superscript notation denotes exponentiation.
+For example, 10 raised to the 100th is notated: 10<sup>100</sup>,
+where 10 is the base and 100 is the exponent.
+In the plain-text rendition of this specification, superscript
+notation is not available and exponentiation therefore is rendered by
+the surrogate notation seen here in the plain-text rendition.
```

Comment:
You are using Unicode [below](https://github.com/lwig-wg/terminology/blob/a413894e1a43f67ef3ac7ac71f7912cafc31afa6/draft-ietf-iotops-7228bis.md?plain=1#L959-L960) already. Why not use it here as well like this "10¹⁰⁰"?